README.md

```diff
@@ -12,21 +12,44 @@ Setting up a fully working Linux system with all needed packages and openHAB rec
 
-*   A set of scripts to set up openHAB on any Debian/Ubuntu based system
-*   Complete **SD-card images pre-configured with openHAB** and many other openHAB- and Hardware-specific preparations, for the *Raspberry Pi* and the *Pine A64* platforms.
+*   a set of scripts to set up openHAB on any Debian based system (Raspbian, Ubuntu)
+*   a complete **SD-card image pre-configured with openHAB** and many other openHAB- and Hardware-specific preparations for all *Raspberry Pi* models.
 
-Closely related to openHABian is the repository [openhab-linuxpkg](https://github.com/openhab/openhab-linuxpkg) which provides linux packages for openHAB. openHABian uses those packages to setup openHAB with some additional configurations.
+## Hardware and OS support
+As of openHABian version 1.5 of the image, all Raspberry Pi models are supported as hardware.
+Anything x86 based may work or not. Anything else ARM based such as ODroids, OrangePis and the like may work or not.
+NAS servers such as QNAP and Synology boxes will not work. Support for PINEA64 was also dropped in this current release.
+We strongly recommend that users choose Raspberry Pi 2, 3 or 4 systems to have 1GB of RAM or more. RPi 1 and 0/0W only have a single CPU core and 512MB. This can be sufficient to run a smallish openHAB setup, but it will not be enough to run a full-blown system with many bindings and memory consuming openHABian features/components such as ZRAM, InfluxDB or Grafana.
+We do not actively prohibit installation on any hardware - including unsupported systems -, but we might skip or deny to install specific extensions such as those memory hungry features named above.
+Supporting hardware means testing every single patch and every release. There are simply too many SBC and HW combinations that maintainers do not have available, or, even if they did, the time to spend on the testing efforts that is required to make openHABian a reliable system.
+Let's make sure you understand the implications of these statements: it means that to run hardware other than RPi 2/3/4 or x86 is **not** _fully_ supported.
+You can try and there's a good chance it'll work out for you. But if you hit any problem related to memory or hardware, you'll be on your own. You are expected **not** to raise these problems as issues on the community forum or on GitHub.
 
-## Installation and Setup
+Going beyond what the RPi image provides, as a manually installed set of scripts, openHABian is supposed to run on all Debian like Linux distributions on either ARM or x86 hardware. Note our automated testing for x86 is based on generic Debian and we take no resposibilities if you run into trouble with a different Linux flavour.
+We recommend to use Ubuntu for x86 hardware and ARMbian for ARM hardware that we don't support.
+Note we expect you to use an up to date distribution: 'Buster' for Raspbian and 'Bionic Beaver' for Ubuntu, these are. To install openHABian on anything older may work or not.
```

Comment:
next Ubuntu LTS version was 20.04 was released (Focal Fossa), but I don't think that we are fully compatible yet.
My recommendation would be not to mention Ubuntu at all...

Reply:
Ok, I've reworded that.
But to destroy hopes, I'm afraid that will not relieve us from a need to support Ubuntu ...

Focal is latest, I believe it is the equivalent of debian testing/bullseye, isn't it ? or that of unstable/sid ?
Did Ubuntu move from bionic to disco (dingo) as the "stable" equivalent of debian buster when they released focal ?

Reply:
Seems to be finally released
https://wiki.ubuntu.com/FocalFossa/ReleaseNotes

Reply:
Yes, I read and caught up on Ubuntu release cycle now.
Focal is the new current long term support version, Bionic was the previous one.
We should take care of both of these (anyone to know a good docker image source for it ?),
Will create an issue to track introduction of support.

But as written: for now, Ubuntu is out of scope.

Thanks for the review.
This is the first time ever for openHABian to make a statement on supported HW and OS.
I think this was overdue. Let's see how it does in terms of reducing false expectations.